samples/matter/light-bulb/readme.rst

```diff
@@ -0,0 +1,150 @@
+
+PWM led (Green Led):
+- Dimmable Light controlled by Matter controller or binded On/Off light switch
+
+Blue LED:
+
+- Blinking slowly indicate that Matter BLE Advertisment is active for Matter Provisioning
+- Fast Blink (100ms) Means that BLE connection is active and Matter Provisioning is ongoing
+
+Red LED:
+
+- Fast Blinking (100ms) indicate Thread network is active for joining provisionmed newtwork
+- 1 Second constant Active Mode indicate That Matter controller subscribed is establis,ent or resumed
+- Single short active indicate Endpoint activation
+
+Sample support Balletto Devkit Joystick buttons
+
+Center Button `SW0`:
+
+- Hold over 3 seconds for doing Matter factoryreset. After Factory reset press DK normal Reset Button for activate BLE advertisment.
+
+Provisioning with Apple Home
+******************************************
+
+1. Flash builded binary to device and press ``Reset`` -button.
+#. After reset you shuold see slowly blinking blue led which indicate that Device is advertisment Matter ready device for provision.
+#. Open ``Apple Home`` application with Iphone or Ipad which is connected to same Wifi with Matter Conttroler and Thread border router.
+#. Click ``+`` -button from app for `Add Accessory`.
+#. Scan QR code:
+
+   .. image:: QRCode.png
+
+#. Click ``Add To Home`` for Start provisioning to Thread neywork and Matter fabric.
+#. You shuold see now fast blinking Blue led when BLE connection is established.
+#. ``Aplle Home`` will warning Uncertificated Acessory so just click ``Add Anyway`` for continue.
+#.  When Provision is ready ``Apple Home`` ask where to add light select a room and click ``Continue``.
+#. Now you can rename a light default is ``Matter Accessory`` name for example ``Matter Light`` and click ``Continue``.
+#. Click ``Continue``
+#. Click at next window ``View at Home`` and you can controll now Green led by Moving bar.
+
+Device is now provisioned and Light controll is possible by `Apple Home` and use with ``Automation``.
+
+Provisioning with chip-tool
+***************************
+
+Using ``chip-tool`` you need to know Thread network active data and Device QR-code payload.
+
+Manual pair code:
+`34970112332`
+
+QR Payload:
+`MT:6FCJ142C00KA0648G00`
+
+How to get Thread network Active data set:
+
+Open terminal to your linux which is running ``Open Thread Border router`` and call next command:
+
+.. code-block:: console
+
+    sudo ot-ctl dataset active -x
+    35060004001fffe00c0402a0f7f8051000112233445566778899aabbccddee00030e4f70656e54687265616444656d6f0410445f2b5ca6f2a93a55ce570a70efeecb000300001a02081111111122222222010212340708fd110022000000000e0800000003601c0000
+    Done
+
+Provision device and update accesslist:
+
+Here is example flow for provision device to with  ``node_id`` 1.
+
+1. Flash builded binary to device and press ``Reset`` button
+#. After reset you shuold see slowly blinking blue led which indicate that Device is advertisment Matter ready device for provision
```

Comment:
indicate -> indicates
is advertisment Matter ready device for provision -> is ready for device provisioning